image_writers_guide/sti_testing.adoc

```diff
@@ -77,37 +46,50 @@ build:
 test:
 	docker build -t $(IMAGE_NAME)-candidate .
 	IMAGE_NAME=$(IMAGE_NAME)-candidate test/run
----
-
-== Basic work flow
-
-This section describes the default work flow you should be using for testing the
-STI image builders:
-
-First build the STI image using 'docker build -t BUILDER_IMAGE_NAME'
-
-1. docker run BUILDER_IMAGE_NAME <verify the usage script is working>
-1. sti build file:///path-to-sample-app BUILDER_IMAGE_NAME OUTPUT_APPLICATION_IMAGE_NAME
-1. <in case you support save-artifacts, execute 2. step again to verify that
-   restoring artifacts works properly>
-1. docker run OUTPUT_APPLICATION_IMAGE_NAME
-1. <verify the container is running and the application is responding>
-
-Executing all the steps above, should be usually enough to tell if the STI
-commands operates properly.
+----
+====
+
+== Basic Testing Workflow
+The `test` script assumes that there is an image to test already built. If required, first build the STI image using:
+
+----
+docker build -t BUILDER_IMAGE_NAME
+----
+
+The following steps describe the default workflow you should be using for testing the STI image builders:
+
+. Verify the `usage` script is working:
++
+----
+docker run BUILDER_IMAGE_NAME
+----
++
+. Build the image:
++
+----
+sti build file:///path-to-sample-app BUILDER_IMAGE_NAME OUTPUT_APPLICATION_IMAGE_NAME
+----
++
+. If you support `save-artifacts`, execute step 2 again to verify that restoring artifacts works properly.
++
+. Run the container:
++
+----
+docker run OUTPUT_APPLICATION_IMAGE_NAME
+----
+. Verify the container is running and the application is responding.
+
+Executing these steps is generally enough to tell if the STI scripts are operating properly.
 
 == Using OpenShift Build for Automated Testing
+Another way you can execute the STI image tests is to use the OpenShift platform itself as a continuous integration system. OpenShift knows how to build the Docker images and it allows a lot of customization.
 
-Another way you can execute the STI image tests is to use OpenShift platform
-itself as a continuous integration system. OpenShift knows how to build the
-Docker images and it allows a lot of customization. To setup a STI image builder
-CI, you have to define a special 'CustomBuild' and use the
-'openshift/sti-image-builder' image. This image is capable to execute all the
-steps mentioned in the section above and produce a new STI builder image.
+To set up a STI image builder CI, define a special `CustomBuild` and use the [sysitem]#openshift/sti-image-builder# image. This image is capable of executing all the steps mentioned in the link:#basic-testing-workflow[Basic Testing Workflow] section and producing a new STI builder image.
```

Comment:
[Suggestion]
This image is capable of executing all the steps mentioned in the link:#basic-testing-workflow[Basic Testing Workflow] section and producing a new STI builder image. ==> This image executes all the steps mentioned in the link:#basic-testing-workflow[Basic Testing Workflow] section and creates a new STI builder image.